dotnet/samples/TeamsChefBot/infra/azure.parameters.json

```diff
@@ -0,0 +1,21 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "resourceBaseName": {
+      "value": "teamschefbot${{RESOURCE_SUFFIX}}"
+    },
+    "botAadAppClientId": {
+      "value": "${{BOT_ID}}"
+    },
+    "botAadAppClientSecret": {
+      "value": "${{SECRET_BOT_PASSWORD}}"
+    },
```

Comment:
Do provision and deploy work? Any preferred way to add ai key to remote deployment?

Reply:
My idea is that user sets AI related envs in `env/.env.dev` or `env/.env.dev.user`, then these envs can be referenced for deploy. Updated `azure.parameters.json` and `azure.bicep`.